demos/connector/neo4j/directed-graphsage-on-cora-neo4j-example.ipynb

```diff
@@ -0,0 +1,1158 @@
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>...</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>5 rows × 1433 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "    0     1     2     3     4     5     6     7     8     9     ...  1423  \\\n",
+       "id                                                              ...         \n",
+       "0      0     0     0     0     0     0     0     0     0     0  ...     0   \n",
+       "1      0     0     0     0     0     0     0     0     0     0  ...     0   \n",
+       "2      0     0     0     0     1     0     0     0     0     0  ...     0   \n",
+       "3      0     0     0     0     1     0     0     0     0     0  ...     0   \n",
+       "4      0     0     0     0     0     0     0     0     0     0  ...     0   \n",
+       "\n",
+       "    1424  1425  1426  1427  1428  1429  1430  1431  1432  \n",
+       "id                                                        \n",
+       "0      0     0     0     0     0     0     0     0     0  \n",
+       "1      0     0     0     0     0     0     0     1     0  \n",
+       "2      0     0     0     0     0     0     0     0     0  \n",
+       "3      0     0     0     0     0     0     0     0     0  \n",
+       "4      0     0     0     0     0     0     0     0     0  \n",
+       "\n",
+       "[5 rows x 1433 columns]"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "node_features = pd.DataFrame(node_data[\"features\"].values.tolist(), index=node_data.index)\n",
+    "node_features.head(5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "At this point, although we do not store all edges, it is still necessary to put all nodes in NetworkX object."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Gnx = nx.DiGraph()\n",
+    "Gnx.add_nodes_from(node_data.index, label=\"paper\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "labels = np.array(node_data[\"subject\"])\n",
+    "\n",
+    "target_encoding = preprocessing.OneHotEncoder(sparse=False, categories=\"auto\")\n",
+    "targets = target_encoding.fit_transform(labels.reshape(-1, 1))\n",
+    "targets = pd.DataFrame(targets, index=node_data.index)"
```

Comment:
There's two `target_encoding` objects in this file, and this `targets` object isn't used. I suspect either this or the `train_targets`/`test_targets` cell can be removed?